chef/cookbooks/postgresql/recipes/server.rb

```diff
@@ -115,17 +115,40 @@
   postgresql_configuration[key] = formatted_value
 end
 
+reloadable_parameters = [
+  "log_line_prefix"
```

Comment:
I think log_truncate_on_ration and log_filename are also reloadable.

Postgresql can be queried for whether a parameter requires a reload or restart with "select context from pg_settings where name = 'log_line_prefix';", maybe at some point in the future we could utilize that.